modules/oe_theme_helper/templates/field--in-page-navigation.html.twig

```diff
@@ -0,0 +1,71 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
```

Comment:
It is copy paste from build/core/themes/stable/templates/field/field.html.twig. Do we need so long description?  Might be ` * @see ./core/themes/stable/templates/field/field.html.twig` is enough?

Reply:
I agree